Makefile

```diff
@@ -436,6 +436,7 @@ docs-test-whitespace:
 	fi
 
 
+ifeq (${DISABLE_TEST_TARGETS},)
```

Comment:
Why test targets needs to be excluded during `make grpc`  call ? 
At first glance it looks it doesn't matter if test targets are disabled or not.

Reply:
The test targets have a bunch of `$(shell go list ./... | grep ...)` that happen at target evaluation time, which makes any `make` invocation noticeably slow while inside docker. Perhaps we should fix this separately, I don't see a good reason why the evaluation is done effectively at the "top level" instead of just happening within each individual rule to begin with.

Reply:
Thanks for explanation.